frontend/src/components/Cases/AccordionItem.js

```diff
@@ -0,0 +1,199 @@
+import AccordionSvg from "./AccordionSvg";
+import { schemaDisplay, copyToClipboard } from "../../utilities/utilFunctions";
+import Svg from "../../assets/svg/Svg";
+
+const AccordionItem = ({ eachCase, implementations, caseImplementation }) => {
+  const seq = eachCase.seq;
+  const description = eachCase.case.description;
+  const schema = eachCase.case.schema;
+  const tests = eachCase.case.tests;
+
+  function result(index, implementation) {
+    var testResult;
+    if (implementation.skipped && implementation.skipped == true) {
```

Comment:
Wouldn't `if (implementation.skipped)` be enough? Or if you really want to check for `true` you could write `if (implementation.skipped === true)`. Same goes for conditions below

Reply:
Made the changes!